gammapy/maps/axes.py

```diff
@@ -2284,81 +2315,84 @@ def reference_time(self):
+        """Return the number of bins in the axis."""
         return len(self.edges_min.flatten())
 
     @property
     def edges_min(self):
-        """Return array of bin edges max values."""
+        """Return the array of bin edges max values."""
         return self._edges_min
 
     @property
     def edges_max(self):
-        """Return array of bin edges min values."""
+        """Return the array of bin edges min values."""
         return self._edges_max
 
     @property
     def edges(self):
-        """Return array of bin edges values."""
+        """Return the array of bin edges values."""
         if not self.is_contiguous:
             raise ValueError("Time axis is not contiguous")
 
         return edges_from_lo_hi(self.edges_min, self.edges_max)
 
     @property
     def bounds(self):
-        """Bounds of the axis (~astropy.units.Quantity)"""
+        """Bounds of the axis as a ~astropy.units.Quantity."""
         return self.edges_min[0], self.edges_max[-1]
 
     @property
     def time_bounds(self):
-        """Bounds of the axis (~astropy.units.Quantity)"""
+        """Bounds of the axis as a ~astropy.units.Quantity."""
         t_min, t_max = self.bounds
         return t_min + self.reference_time, t_max + self.reference_time
 
     @property
     def time_min(self):
-        """Return axis lower edges as Time objects."""
+        """Return axis lower edges as `~astropy.time.Time` object."""
         return self._edges_min + self.reference_time
 
     @property
     def time_max(self):
-        """Return axis upper edges as Time objects."""
+        """Return axis upper edges as a `~astropy.time.Time` object."""
         return self._edges_max + self.reference_time
 
     @property
     def time_delta(self):
-        """Return axis time bin width (`~astropy.time.TimeDelta`)."""
+        """Return axis time bin width as a `~astropy.time.TimeDelta` object."""
         return self.time_max - self.time_min
 
     @property
     def time_mid(self):
-        """Return time bin center (`~astropy.time.Time`)."""
+        """Return time bin center as a `~astropy.time.Time` object."""
         return self.time_min + 0.5 * self.time_delta
 
     @property
     def time_edges(self):
-        """Time edges"""
+        """Time edges as a `~astropy.time.Time` object."""
         return self.reference_time + self.edges
 
     @property
     def time_format(self):
+        # TODO: The docsttring for time format and its setter are not showing up in the docs.
+        """The time format to use for the axis."""
         return self._time_format
 
     @time_format.setter
     def time_format(self, val):
+        """The time format to use for the axis."""
         if val not in ["iso", "mjd"]:
             raise ValueError(f"Invalid time_format: {self.time_format}")
         self._time_format = val
 
     @property
     def as_plot_xerr(self):
-        """Plot x error"""
+        """Return x errors as if plotting."""
```

Comment:
You've used this wording `as if plotting` a number of times, but it is not so clear.
My suggestion would be `for plotting`